website/config.toml

```diff
@@ -1,5 +1,5 @@
 baseURL = "/"
-languageCode = "en-us"
+# languageCode = "en-us"
```

Comment:
This line was copied from an example config, and `<html lang="en" class="no-js">` seems to indicate that the language code is coming from `[languages.en]`. Lets just delete it.
```suggestion
```

Reply:
will do.